vitality/events.py

```diff
@@ -0,0 +1,13 @@
+import datetime
+class Event:
+    def __init__(self, _id: str, creator_id: str, date: datetime, title: str, description: str,):
+        """Constructor for Workout class."""
+        self._id = _id
+        self.creator_id = creator_id
+        self.date = date
+        self.title = title
+        self.description = description
+
```

Comment:
Add a `__repr__` function to return a formatted string of the object. 

As an example look at the Trainee class.